internals/secrethub/generate.go

```diff
@@ -28,6 +28,7 @@ var (
 	ErrMinFlagInvalidInteger     = errGenerate.Code("min_flag_invalid_int").ErrorPref("second part of --min flag is not an integer: %s")
 	ErrCharsetSizeNonPositive    = errGenerate.Code("charset_size_non_positive").Error("charset size must be > 0")
 	ErrFlagsMutuallyExclusive    = errGenerate.Code("include_exclude_flags_mutually_exclusive").ErrorPref("the following flags are mutually exclusive: --include %s, --exclude %s")
+	ErrInvalidMinFlag            = errGenerate.Code("min_flag_invalid").ErrorPref("min flag is invalid: %s")
```

Comment:
Should the error explain what is a valid value for the min flag? That way, the user can correct their error.